docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+# Contributing
+
+### Issues, Bugs, Feature or Component Requests
+
+While we don't accept issues from people outside the Adevinta Platform to this repo currently, we
+welcome [discussions and questions here](https://github.com/adevinta/spark-android/discussions/categories/general)!
+
+## If you're part of the Adevinta Platform ⬇️
+
+### Requirements
+
+#### [git-lfs](https://git-lfs.com/)
+
+This repository uses [cashapp/paparazzi](https://github.com/cashapp/paparazzi) for screenshot
+testing purposes.  
+If you introduce visible changes, you'll likely have to update screenshots files for the test to
+pass.
+
+```bash
+# Install
+brew install git-lfs
+# Check
+git lfs install
+```
+
+#### [java 17](https://github.com/adevinta/spark-android/issues/74)
+
+We use AGP 8.1.0 which requires developers to use JDK 17 on Gradle JDK
+If you're on macOS, you can install it with [brew](https://github.com/mdogan/homebrew-zulu/)
```

Comment:
I would not encourage people to install external brew taps. Instead we should redirect them to the official source of JDK zulu (and maybe add the link to mdogan's tap)